masterlist.yaml

```
@@ -9386,6 +9386,42 @@ plugins:
       - crc: 0x4D5B6393
         util: 'SSEEdit v4.0.4b'
 
+  - name: 'Dead Man''s Dread Distribution.*\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/35163/'
+        name: 'Pirate''s Life for Me - A Creation Club Integration Mod'
+  - name: 'Dead Man''s Dread Distribution( - Pirates Wield Cutlasses Patch)?\.esp'
+    tag:
+      - Invent.Add
+      - Invent.Remove
+      - NPC.DefaultOutfit
+  - name: 'Dead Man''s Dread Distribution.esp'
+    msg:
+      - <<: *patchIncluded
+        subs: [ 'Finding Velehk Sain' ]
+        condition: 'active("Finding_VelehkSain.esp") and not active("Dead Man''s Dread Distribution - Finding Velehk Sain Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Pirates Wield Cutlasses' ]
+        condition: 'active("Pirates Wield Cutlasses.esp") and not active("Dead Man''s Dread Distribution - Pirates Wield Cutlasses Patch.esp")'
+    tag:
+      - Actors.DeathItem
+      - Names
+      - Sound
+    dirty:
+      - <<: *quickClean
+        crc: 0x5DF95909
+        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 3
+  - name: 'Dead Man''s Dread Distribution - Pirates Wield Cutlasses Patch.esp'
+    tag: [ Delev ]
+    clean:
+      - crc: 0x9CC84024
+        util: 'SSEEdit v4.0.4b'
+  - name: 'Dead Man''s Dread Distribution - Finding Velehk Sain Patch.esp'
+    clean:
+      - crc: 0xC1A20E44
+        util: 'SSEEdit v4.0.4b'
```

These patches should be sorted alphabetically:

```suggestion
  - name: 'Dead Man''s Dread Distribution - Finding Velehk Sain Patch.esp'
    clean:
      - crc: 0xC1A20E44
        util: 'SSEEdit v4.0.4b'
  - name: 'Dead Man''s Dread Distribution - Pirates Wield Cutlasses Patch.esp'
    tag: [ Delev ]
    clean:
      - crc: 0x9CC84024
        util: 'SSEEdit v4.0.4b'
```